grizli/tests/test_jwst.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for JWST imaging and spectra, including 
+"""
+import os
+import glob
+import unittest
+
+import numpy as np
+from .. import utils, prep, multifit, fitting, GRIZLI_PATH
+
+
+class JWSTFittingTools(unittest.TestCase):
+    
+    def test_multibeam(self):
+        """
+        Can we initialize a multibeam file?
+        """
+        path = os.path.dirname(utils.__file__)
+        print(path)
+        beams_file = '/Users/victoriastrait/Desktop/grizli_test_data/jw01324001001_01243.beams.fits'
```

Comment:
This will fail in the online tests. You could make a test to quit gracefully if not os.path.exists(beams_file), or if the file isn't too big you could put it in grizli/tests/data in the repo. If you do the latter, it would be good to call it something like niriss_jw01....beams.fits.